.gitignore

```diff
@@ -1,23 +1,27 @@
-# Compiled class file
-*.class
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
Can you explain the changes to the .gitignores? These changes would ideally have their own issue

Reply:
I usually open the VS code at the root folder, which will generate .vs and other cache folders automatically. And original gitignore from this repo files are not able to filter the cache folders.